static-location.js

```diff
@@ -1,13 +1,4 @@
-// Unfortunately we cannot import these from use-location, because
-// that will also pull in react, use-sync-external-store, and then
-// monkeypatch `history` *again* in the generated build files!
-const relativePath = (base = "", path = location.pathname) =>
-  !path.toLowerCase().indexOf(base.toLowerCase())
-    ? path.slice(base.length) || "/"
-    : "~" + path;
-
-const absolutePath = (to, base = "") =>
-  to[0] === "~" ? to.slice(1) : base + to;
+import { absolutePath, relativePath } from "./path-functions.js";
```

Comment:
Would be nice to remove the word `function` from the module name. Maybe, `paths.js` or `base-path.js`?